dist/api_data.py

```diff
@@ -45,6 +45,43 @@ def __ge__(self, other):
 common_runtime_config = [
     Config('app_metadata', '', r'''
         application-owned metadata for this object'''),
+    Config('assert', '', r'''
+        enable enhanced checking. ''',
+        type='category', subconfig= [
+        Config('write_timestamp', 'off', r'''
+            verify that commit timestamps are used per the configured
+            \c write_timestamp_usage option for this table''',
+            choices=['on', 'off']),
```

Comment:
"always" and "never" would be better since they match those for the read_timestamp option (even though they mean different things).

Reply:
I tend to agree with @tetsuo-cpp that if they mean different things, it helps the users discover that. That said, please alphabetize the options.

Reply:
I'll stand down on this one, but I did have further motivation.

I think "always" and "never" are better for future extension. For example "on", "prepare", "off" are hard to interpret - as a user I need to ask "do I need to turn it `on` to get `prepare`?"

Reply:
9247c41